### ISSUE TITLE
Friendly suggestion to improve accessibility

### ISSUE THREAD

**Issue by TTomczek** [TTomczek/studygroupmap#28]
Hi there! I really appreciate the work done on this project. I wanted to kindly point out that accessibility features seem to be missing. It would be fantastic to have improved accessibility to make the project more inclusive for everyone. Thanks for considering this!
